lib/bundler/vendor/uri/lib/uri/common.rb

```diff
@@ -498,7 +498,7 @@ def self._decode_uri_component(regexp, str, enc)
   # each +key+/+value+ pair is converted to one or more fields:
   #
   # - If +value+ is
-  #   {Array-convertible}[rdoc-ref:implicit_conversion.rdoc@Array-Convertible+Objects],
+  #   {Array-convertible}[rdoc-ref:ruby/implicit_conversion.rdoc@Array-Convertible+Objects],
```

Comment:
Vendored files are excluded not to duplicate the documents.

Reply:
Excluded the change